dao/dao.go

```diff
@@ -67,22 +70,213 @@ func New(s server, a auth, opts *opts) (*DAO, error) {
 }
 
 // Save stores data into table in clickhouse database
-func (p *DAO) Save(data string) error {
-	if data == "" {
-		return fmt.Errorf("clickhouse invalid data for table %s: %s", p.table, data)
-	}
+// nolint: funlen
+func (p *DAO) Save(theEvent beacon.RumEvent) error {
 	query := fmt.Sprintf(
-		"INSERT INTO %s SETTINGS input_format_skip_unknown_fields = true FORMAT JSONEachRow %s",
+		`INSERT INTO %s (
```

Comment:
@ivan-stankov-golang is it possible still to keep this format o the insert statement?

```
"INSERT INTO %s SETTINGS input_format_skip_unknown_fields = true FORMAT JSONEachRow %s"
```

It is really ClickHouse specific but the`input_format_skip_unknown_fields = true` setting makes it much more flexible so we don't need to list the exact fields we want to insert.